### ISSUE TITLE
Last libcurl does not compile with pkg-config v0.26

### ISSUE THREAD

**Issue by e2iplayer** [curl/curl#6073]
There is problem in cofigure with last libcurl:
variable addcflags is set as follow
```
      addcflags=`
    if test -n "$wolfpkg"; then
      PKG_CONFIG_LIBDIR="$wolfpkg"
      export PKG_CONFIG_LIBDIR
    fi

        $PKGCONFIG --cflags-only-I wolfssl`
```

pkg-config v0.26 returns include path with 2 spaces at the end of path
So, variable contains for example: "-I/usr/include  "
Then:
```
CPPFLAGS="$addcflags/wolfssl $CPPFLAGS"
```

This couse configure error because the CPPFLAGS is set as:
 "-I/usr/include  /wolfssl " 
```
Add -I/usr/include  /wolfssl to CPPFLAGS
```

and gcc say:
gcc: error: /wolfssl: No such file or directory

**Comment by bagder** [comment]
I believe this was fixed already in #5848 and commit 510d98157f21d.

The next release is due to ship in... about 8 hours! :grinning:

**Comment by e2iplayer** [comment]
Yes, it looks like same issue. 
I searched but I did not found it...

I am sorry, for the disturbance and thank you for this great project.
